src/dotnet/commands/dotnet-sln/LocalizableStrings.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Sln
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET modify solution file(s) command";
+
+        public const string AppDescription = "Command to add, remove and list projects from the solution (SLN) file.";
+
+        public const string AppHelpText = "Projects to add to the solution.";
+
+        public const string CmdSlnFile = "<SLN_FILE>";
+
+        public const string CmdSlnFileText = "Solution file to operate on. If not specified, the command will search the current directory for one.";
+
+        public const string AddSubcommandHelpText = "Add a specified project to the solution.";
```

Comment:
I need the AppFullName and AppDescription for each of the subcommands as well. You have one for the top-level command, but I need versions for each subcommand that appears for example when you do dotnet sln add -h